integration-tests/src/test/scala/tech/beshu/ror/integration/suites/SearchApiSuite.scala

```diff
@@ -204,13 +271,69 @@ object SearchApiSuite {
   private def nodeDataInitializer(): ElasticsearchNodeDataInitializer = (esVersion, adminRestClient: RestClient) => {
     val documentManager = new DocumentManager(adminRestClient, esVersion)
     val indexManager = new IndexManager(adminRestClient, esVersion)
+    val templateManager = new IndexTemplateManager(adminRestClient, esVersion)
+    val dataStreamManager = new DataStreamManager(adminRestClient, esVersion)
+
+    createDataStream(templateManager, dataStreamManager, "test_logs_ds", "test_logs_ds_index_template")
+    createDocsInDataStream(documentManager, indexManager, "test_logs_ds", 5)
 
-    createDataStreamWithAlias(indexManager, documentManager)
+    indexManager
+      .updateAliases(
+        AliasAction.Add(index = "test_logs_ds", alias = "alias_ds"),
+      )
+      .force()
+
+    createOldFashionedDataStream(indexManager, documentManager)
     createRealLifeTestsDocumentsAndAliases(indexManager, documentManager)
   }
 
-  private def createDataStreamWithAlias(indexManager: IndexManager,
-                                        documentManager: DocumentManager) = {
+  private def createDataStream(adminTemplateManager: IndexTemplateManager,
+                               adminDataStreamManager: DataStreamManager,
+                               dataStreamName: String,
+                               indexTemplateName: String): Unit = {
+    adminTemplateManager.createTemplate(indexTemplateName, indexTemplate(dataStreamName)).force()
+    adminDataStreamManager.createDataStream(dataStreamName)
+  }
+
+  private def indexTemplate(dataStreamName: String) = ujson.read(
+    s"""
+       |{
+       |  "index_patterns": ["$dataStreamName*"],
+       |  "data_stream": { },
+       |  "priority": 500,
+       |  "template": {
+       |    "mappings": {
+       |      "properties": {
+       |        "@timestamp": {
+       |          "type": "date",
+       |          "format": "date_optional_time||epoch_millis"
+       |        },
+       |        "message": {
+       |          "type": "wildcard"
+       |        }
+       |      }
+       |    }
+       |  }
+       |}
+       |""".stripMargin
+  )
+
+  private def createDocsInDataStream(adminDocumentManager: DocumentManager, indexManager: IndexManager, streamName: String, count: Int): Unit = {
+    def format(instant: Instant) = instant.toString
+
+    def createDoc(c: Int): Unit = {
+      val doc = ujson.read(s"""{ "message":"test$c", "@timestamp": "${format(Instant.now())}"}""")
```

Comment:
ah, here. Maybe instead of "test" it'd be better to have the "message" string in here?